lua/nvim-biscuits/init.lua

```diff
@@ -12,11 +12,18 @@ local ts_parsers = require('nvim-treesitter.parsers')
 local ts_utils = require('nvim-treesitter.ts_utils')
 local nvim_biscuits = {}
 
+local should_render_biscuits = true
+
 local make_biscuit_hl_group = function(lang) return 'BiscuitColor' .. lang end
 
 nvim_biscuits.decorate_nodes = function(bufnr, lang)
     if config.get_language_config(final_config, lang, "disabled") then return end
 
+    if not should_render_biscuits then
+        vim.api.nvim_buf_clear_namespace(bufnr, 0, 0, -1)
```

Comment:
So when I try toggle `biscuits`, it disables all virtual text. I think this line is the cause. Maybe you can see [this](https://github.com/lukas-reineke/indent-blankline.nvim/blob/1a61a0bb0e67675b0a4cf9ffbfca100e498a1d04/lua/indent_blankline/commands.lua#L39) as reference. And sorry for the late reply.

Reply:
Its because of the highlight group I am using. I can change that.
It is also an issue in another PR, so I will do that first. I will let you know when I have done that. Thinking Sunday seems likely.

Reply:
Ok thank you

Reply:
I have the fix merged in now. Let me know if that helps.